resources/assets/js/components/EventDescription.vue

```diff
@@ -9,8 +9,6 @@
   </CollapsibleSection>
 </template>
 <script>
-import { DATE_FORMAT } from '../constants'
-import moment from 'moment'
```

Comment:
I discovered yesterday that moment isn't being actively evolved.  At some point we should switch to dayjs, as it's a lot smaller.  That's a tech debt task for much later, if/when we move to a fully VueJS app.  Do we have a place to dump things like that as we go?

Reply:
A ticket in Jira in the 'Platform - Infrastructure Issues' epic would be a good place to log tech debt - although I know Laurence is keen not to have tickets created for 'nice to haves', only for things we do actively plan on doing (at some point), because the backlog gets a bit unmanageable otherwise.

At the same time, it's nice not to lose ideas... perhaps we could have a tech debt doc for the nice to haves?  And if something keeps on popping up repeatedly, then it's more than a nice to have?

Reply:
Yes, a tech debt doc - probably better for us to have the raw stuff somewhere so that we can sort through to turn it into Jira tickets as and when.  Vue refactoring is the key tech debt task for now anyway.